### PR TITLE
linux-firmware: update to 20230210

### PR DESCRIPTION
**Pull request by absrd**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/linux-firmware/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'linux-firmware'
 pkgname=linux-firmware
-version=20221109
+version=20221214
 revision=1
 depends="${pkgname}-amd>=${version}_${revision} ${pkgname}-network>=${version}_${revision}"
 short_desc="Binary firmware blobs for the Linux kernel"
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="See /usr/share/licenses/${pkgname}"
```
[the remark this answers is not in the document]
Comment (absrd) [reply srcpkgs/linux-firmware/template:8]: Well, I'm not going to bother with listing all the licenses from the license files in the package.

Feel free to close the PR if it's required.

[the remark this answers is not in the document]
Comment (icp1994) [reply srcpkgs/linux-firmware/template:8]: I think it's allowed to do something like `license="custom:linux-firmware" in such cases - maybe someone can correct/confirm me on that.

[the remark this answers is not in the document]
Comment (brxken128) [reply srcpkgs/linux-firmware/template:8]: > I think it's allowed to do something like `license="custom:linux-firmware" in such cases - maybe someone can correct/confirm me on that.

It seems to be allowed, [chromium-widevine does something similar.](https://github.com/void-linux/void-packages/blob/7dd7a5b371a82f5e6605c082d3dcf50cbaf8baee/srcpkgs/chromium-widevine/template)

**Comment by dataCobra** [comment]
Hey @absrd,

could you update to the current release?

I'll try to get attention to this PR so it gets merged. :+1:

**Comment by omnisci3nce** [comment]
I've just built a new PC with an AMD 7900xt graphics card and ran into the issues specified here: https://discussion.fedoraproject.org/t/amdgpu-doesnt-seem-to-function-with-navi-31-rx-7900-xtx/72647/4
I've manually downloaded the latest `linux-firmware` to copy the `/amdgpu` folder over but this being updated would be epic for new/noob voidlinux users like me. (Sorry for not contributing directly via this comment. Thanks everyone for keeping the packages up-to-date, I'd love to contribute back soon)

**Comment by brxken128** [comment]
> I've just built a new PC with an AMD 7900xt graphics card and ran into the issues specified here: https://discussion.fedoraproject.org/t/amdgpu-doesnt-seem-to-function-with-navi-31-rx-7900-xtx/72647/4
> I've manually downloaded the latest `linux-firmware` to copy the `/amdgpu` folder over but this being updated would be epic for new/noob voidlinux users like me. (Sorry for not contributing directly via this comment. Thanks everyone for keeping the packages up-to-date, I'd love to contribute back soon)

Yeah, I ran into this issue back in December also (except the card was far too new back then - broken drivers and such were expected).

Fully in agreement that this'd help out!

**Pull request merged**